### ISSUE TITLE
Repository.get_issues() returns PRs as well as issues

### ISSUE THREAD

**Issue by ckelleyRH** [PyGithub/PyGithub#2010]
I would expect `get_issues()` to return only issues, but it appears to also return PRs. I have the following stub test:
``` 
g = Github("my token")
jss_repo = g.get_repo("dogtagpki/jss")
i = 0
for issue in jss_repo.get_issues():
    print(issue.title)
    i += 1
print(i)
```

<details>
<summary>which produces the following result:</summary>
<p>

```
Replace jaxb XML with jackson JSON serialisation
JSSEngine.queryEnabledCipherSuites() generates unknown cipher warnings
NPE in Buffer.WriteCapacity()
Autoformat org.mozilla.jss.ssl using uncrustify
Autoformatting using uncrustify
Autoformat tools
Autoformat examples
Autoformat org.mozilla.jss.tests
Autoformat org.mozilla.jss.util
Autoformat org.mozilla.jss.ssl.javax
Autoformat org.mozilla.jss.ssl
Autoformat org.mozilla.jss.SecretDecoderRing
Autoformat org.mozilla.jss.provider.javax.crypto
Autoformat org.mozilla.jss.provider.java.security
Autoformat org.mozilla.jss.pkix.primitive
Autoformat org.mozilla.jss.pkix.crmf
Autoformat org.mozilla.jss.pkix.cms
Autoformat org.mozilla.jss.pkix.cmc
Autoformat org.mozilla.jss.pkix.cmmf
Autoformat org.mozilla.jss.pkix.cert
Autoformat org.mozilla.jss.pkcs7
Autoformat org.mozilla.jss.pkcs12
Autoformat org.mozilla.jss.pkcs11.attrs
Autoformat org.mozilla.jss.pkcs11
Autoformat org.mozilla.jss.pkcs10
Autoformat org.mozilla.jss.nss
Autoformat org.mozilla.jss.netscape.security.x509
Autoformat org.mozilla.jss.netscape.security.util
Autoformat org.mozilla.jss.netscape.security.pkcs
Autoformat org.mozilla.jss.netscape.security.pkcs
Autoformat .java and .c files in org.mozilla.jss
Autoformat .java, .c, .h org.mozilla.jss.netscape.security.extensions
Autoformat .java, .c, .h files in org.mozilla.jss.crypto
Replace tabs with spaces
pkispawn fails to create a CA with a YubiHSM, in JSS - org.mozilla.jss.NoSuchTokenException: No such token: YubiHSM
Add support for loading keys from .p12
Switch to checking for AES_KEY_WRAP by default
Only provision OAEP parameters when non-NULL
Fix ipv6 issues ci
Add ip link show to CI
Unable to create SSL connection with SoftHSM
Added oaep jss test
[bot] auto-update gh-pages from source
Fully remove `SSL_` and `SSL3_` ciphersuites
Review LDAP domain name parsing
Mavenize JSS
Unable to create SSL connection for Cassandra
Unable to create SSL connection for PostgreSQL
[WIP] Optimized JSSEngine
Basic tests for PKCS#10
JSSEngine - Initial ALPN support
Add support for AES GCM
API for managing NSS modules
Clean up source file main methods
Validate against JDK test suites
Combine with netscape.security
javax interface check
Key Agreement
58
```

</p>
</details>

TL;DR, there are 58 items listed: 16 issues and 42 PRs from https://github.com/dogtagpki/jss

Is this the expected behaviour, or is it a bug?

**Comment by s-t-e-v-e-n-k** [comment]
From https://docs.github.com/en/rest/reference/issues#list-repository-issues

`Note: GitHub's REST API v3 considers every pull request an issue, but not every issue is a pull request. For this reason, "Issues" endpoints may return both issues and pull requests in the response.`

**Comment by ckelleyRH** [comment]
@s-t-e-v-e-n-k thanks for pointing me to the right place in the docs, googling at gone midnight is a fool's errand :laughing:
